tests/Rector/ClassMethod/ParamConverterAttributeToMapEntityAttributeRector/Fixture/replace_paramconverter.php.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Rector\Symfony\Tests\Rector\ClassMethod\ParamConverterAttributeToMapEntityAttributeRector\Fixture;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ReplaceParamConverterWithMapping
+{
+    #[Route('/blog/{blogSlug}', name: 'blog_detail')]
+    #[ParamConverter('blog', options: ['mapping' => ['blogSlug' => 'slug']])]
+    public function convert(
+        Request $request,
+        stdClass $blog,
+    ): Response
+    {
+        return new Response();
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Symfony\Tests\Rector\ClassMethod\ParamConverterAttributeToMapEntityAttributeRector\Fixture;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ReplaceParamConverterWithMapping
+{
+    #[Route('/blog/{blogSlug}', name: 'blog_detail')]
+    public function convert(
+        Request $request,
+        #[\Symfony\Bridge\Doctrine\Attribute\MapEntity(mapping: ['blogSlug' => 'slug'])] stdClass $blog
```

Comment:
it needs separate line as reported in failing test:

```suggestion
        #[\Symfony\Bridge\Doctrine\Attribute\MapEntity(mapping: ['blogSlug' => 'slug'])]
        stdClass $blog,
```

Reply:
So you are saying it is impossible to have the attribute on the same line as the parameter? Changing the test feels like cheating 😅

Reply:
That's Rector convension to not inline it https://github.com/rectorphp/rector-src/blob/8ef808705cd8f467602f2411ec831552648295ae/src/PhpParser/Printer/BetterStandardPrinter.php#L469